Add October 2022 newsletter (#976)
USRSE/usrse.github.io: _posts/newsletters/2022-11-04-newsletter.md

---
layout: post
title: US-RSE October 2022 Newsletter
subtitle: October 2022 
category: newsletter
tags: [newsletter, supercomputing]
---

> In this monthly newsletter, we share recent, current, and planned activities of the US-RSE Association, and related news that we think is of interest to US-RSE members. Newsletters are also available on our [website](https://us-rse.org/newsletters/) alongside the growing resources and information on the US-RSE Association. To receive our newsletter, join US-RSE [here](https://us-rse.org/join/).

Reviewer: ```suggestion
In this monthly newsletter, we share recent, current, and planned activities of the US-RSE Association, and related news that we think is of interest to US-RSE members. Newsletters are also available on our [website](https://us-rse.org/newsletters/) alongside the growing resources and information on the US-RSE Association. To receive our newsletter, [join US-RSE](https://us-rse.org/join/).
```

Reviewer 2: +1

In this issue:

* [Steering Committee Elections](#elections)
* [First US-RSE Conference in 2023!](#conference2023)
* [RSE-HPC-2022 Workshop at SC22](#sc22)
* [US-RSE SC22 Booth](#scbooth)
* [ADSA/US-RSE Career Workshop](#adsa-usrse)
* [Help Create a Job Description Archive](#jobarchive)
* [Steering Committee Recap](#sc-update)
* [Community News](#news)
* [Funders Discussion Series](#funders-talk)
* [RSE Stories Podcast - Now Developer Stories!](#podcast)
* [November Community Call](#community-call)
* [Interesting Events](#events)
* [Interesting Reads, Videos or Podcasts](#reads)
* [US-RSE Education and Training working group](#education)
* [US-RSE Outreach](#outreach)
* [US-RSE Financials](#financials)
* [Get Involved](#involved)
* [Recent Job Postings](#jobs)

    
    
    
<a name="elections"></a>
# **Steering Committee Elections**
    

> The [yearly election](https://us-rse.org/about/election/) cycle for the Steering Committee is underway.  Information on the candidates will be posted soon.  **Voting will occur December 2-9, 2022.**  US-RSE members were sent an email with the subject "US-RSE Membership Verification" in October confirming their membership.  Everyone who is a [member of US-RSE](https://us-rse.org/join) as of November 20th can vote in the election.  Question about your membership status?  Email membership@us-rse.org.

Reviewer: ```suggestion
The [yearly election](https://us-rse.org/about/election/) cycle for the Steering Committee is underway.  Information on the candidates will be posted soon.  **Voting will occur December 2-9, 2022.**  US-RSE members were sent an email with the subject "US-RSE Membership Verification" in October confirming their membership.  Everyone who is a [member of US-RSE](https://us-rse.org/join) as of November 20th can vote in the election.  Question about your membership status?  Email [membership@us-rse.org](mailto:membership@us-rse.org).
```

Reviewer 2: +1

Reviewer: or could make "email" the link text to match below

Reviewer: And another link below uses the name of the recipient as link text ....

 * The first US-RSE Conference in October 2023: more details in this newsletter
 * What funding US-RSE needs to grow as a community and what paid positions US-RSE needs to be successful, such as a Community Manager
 * Improving the Code of Conduct reporting and response processes (there's a working group for anyone interested)
 * Distribution of US-RSE swag and options for opening a US-RSE store again this year for fundraising
 * Community events and virtual talk series
 * Governance of US-RSE, including issues such as: the structure of, and communication between, working groups and whether an advisory board separate from the Steering Committee would be useful
    

<a name="news"></a>
# **Community News**

* A new RSE association has been formed in Asia! We in US-RSE are looking forward to international collaboration with the [RSE Asia Association](https://rse-asia.github.io/RSE_Asia/)!
* [ACCESS](https://www.nsf.gov/news/special_reports/announcements/042222-access.jsp) - the successor of XSEDE - started, and you can apply with research projects for large HPC resources. [https://support.access-ci.org/matchplus](https://support.access-ci.org/matchplus)

> * [CI CoE: SGX3 - A Center of Excellence to Extend Access, Expand the Community, and Exemplify Good Practices for CI Through Science Gateways](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2231406) was just funded for five years and is extending the work that started with the Science Gateways Community Institute. [SGX3](https://sciencegateways.org/) guides the annual Gateways conference series, offers Focus Weeks that provide training on sustainability of research projects, provides Gateways Central that will be a focal point of exchanging knowledge, experience and technology of science gateway solutions and frameworks and Blueprint Factories target different topics and technologies.

Reviewer: Not sure if a late comma or more is needed here, by the end of the last sentence I am losing track of the grammar and significance. Does SGX3 provide Blueprint Factories or are we just saying something about what they do? Though presumably the solutions and frameworks also target different topics and technologies? Not a hard refusal, just a "you're losing me here".

Reviewer 3: Yes - It got a bit much. How would be "[SGX3](https://sciencegateways.org/) guides the annual Gateways conference series, offers Focus Weeks that provide training on sustainability of research projects, provides Gateways Central that will be a focal point of exchanging knowledge and technology of science gateway solutions  as well as creates Blueprint Factories that target different topics and technologies."

Author: I think that's good, but I'd add a comma and use "and" instead of "as well as." I'll make the suggested change

> * [CI CoE: SGX3 - A Center of Excellence to Extend Access, Expand the Community, and Exemplify Good Practices for CI Through Science Gateways](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2231406) was just funded for five years and is extending the work that started with the Science Gateways Community Institute. [SGX3](https://sciencegateways.org/) guides the annual Gateways conference series, offers Focus Weeks that provide training on sustainability of research projects, provides Gateways Central that will be a focal point of exchanging knowledge, experience and technology of science gateway solutions and frameworks and Blueprint Factories target different topics and technologies.

Author: ```suggestion
* [CI CoE: SGX3 - A Center of Excellence to Extend Access, Expand the Community, and Exemplify Good Practices for CI Through Science Gateways](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2231406) was just funded for five years and is extending the work that started with the Science Gateways Community Institute. [SGX3](https://sciencegateways.org/) guides the annual Gateways conference series, offers Focus Weeks that provide training on sustainability of research projects, provides Gateways Central that will be a focal point of exchanging knowledge and technology of science gateway solutions, and creates Blueprint Factories that target different topics and technologies.
```

Author: @sandragesing is this good/correct?

Author: I'm going to accept this and we can fix it later!

    * [Second Day](https://www.youtube.com/watch?v=7Byp6BS32oM&ab_channel=USResearchSoftwareEngineerAssociation)
* Matthew Greenhouse: [The James Webb Space Telescope Mission](https://www.youtube.com/watch?v=b3hDUhqW8ls&ab_channel=ScienceGatewaysCommunity), Video of keynote at [Gateways 2022](https://sciencegateways.org/gateways2022)
* RSECon2022 took place in Newcastle, UK, in September 2022. Here are some of the recordings. More will become available soon.
    * [Oracle keynote on Gravitational Waves](https://www.youtube.com/watch?v=0P_yAqhbZVA)
    * [SocRSE AGM](https://www.youtube.com/watch?v=TLdTmtQ_VNI)
    * [Keynote from Neil Ferguson](https://www.youtube.com/watch?v=u-4fXU0G6Ig)
    * [RSE Funding –  eCSE Panel](https://www.youtube.com/watch?v=Db7F9_aVr5U)
    * [Christopher Woods on Funding Software Maintenance](https://www.youtube.com/watch?v=JuCBQH7pcxU)
 * Daniel S. Katz: [How groups can immediately benefit when new members join](https://danielskatzblog.wordpress.com/2022/10/31/new-group-members/) (blog post)

<a name="elev-wg"></a>
# **Elevating RSEs Working Group**

> Many of us have similar stories in regards to recognizing the criticality of our roles in supporting modern research endeavors, yet struggle to gain the recognition of that amongst campus leadership, which oftentimes still thinks in terms of more traditional university roles such as faculty and students.  The aim of the Elevating RSEs working group is to help all of us make this clearer at our various organizations.  A key aspect of this is making it very clear to researchers, as well as funding agencies, the scientific discoveries that we are enabling, the scholarship that could not be done otherwise.  As a first step we are soliciting from the RSE community stories of such discoveries that were RSE enabled!  If you have such a story please fill out this brief form [here](https://docs.google.com/forms/d/1zCf4CShRvK3ec1IJasTgGi0_a568gPVRZPMlv3nmlZY).  From these stories we will be doing three things in the short term:

Reviewer: ```suggestion
Many of us have similar stories in regards to recognizing the criticality of our roles in supporting modern research endeavors, yet struggle to gain the recognition of that amongst campus leadership, which oftentimes still thinks in terms of more traditional university roles such as faculty and students.  The aim of the Elevating RSEs working group is to help all of us make this clearer at our various organizations.  A key aspect of this is making it very clear to researchers, as well as funding agencies, the scientific discoveries that we are enabling, the scholarship that could not be done otherwise.  As a first step we are soliciting from the RSE community stories of such discoveries that were RSE enabled!  If you have such a story please fill out [this brief form](https://docs.google.com/forms/d/1zCf4CShRvK3ec1IJasTgGi0_a568gPVRZPMlv3nmlZY).  From these stories we will be doing three things in the short term:
```

- highlighting one story in our newsletter each month,
- compiling a slide deck ([here](https://docs.google.com/presentation/d/1s_5EhuxKm2aJbvJ4X-Lb2BXOputs5GzZhJ4_ion2Nfg/edit#slide=id.p)) with these stories that we can pull slides from when we talk about the importance of RSEs with our colleagues, collaborators, and leadership, and
- collecting outreach channels we should aim to include for our newsletters and other outreach activities.

> Longer term plans include other activities along this front such as an RSE conference where submitted papers are peer reviewed on both the science and the RSE aspects of the effort.  If you have other ideas and are interested in being part of this working group, please join the [#wg-elevate-rse channel](https://usrse.slack.com/archives/C03DW50KY80) in the USRSE slack. 

Reviewer: ```suggestion
Longer-term plans include other activities along this front such as an RSE conference where submitted papers are peer reviewed on both the science and the RSE aspects of the effort.  If you have other ideas and are interested in being part of this working group, please join the [#wg-elevate-rse channel](https://usrse.slack.com/archives/C03DW50KY80) in the USRSE slack. 
```

<a name="outreach"></a>
# **US-RSE Outreach**

> Please join us in Dallas, Texas, after the RSEs in HPC Workshop for our [US-RSE Social Event](https://us-rse.org/events/2022/2022-11-sc22-social/). Check the event page or Slack for details on the location!

Reviewer: ```suggestion
Please join us in Dallas, Texas, after the RSEs in HPC Workshop, for our [US-RSE Social Event](https://us-rse.org/events/2022/2022-11-sc22-social/). Check the event page or Slack for details on the location!
```
